ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityForm.ts

```diff
@@ -50,11 +50,11 @@ export class ActivityForm {
         return this.sections.filter(section => section.visible).length;
     }
 
-    private getAllSections(): Array<ActivitySection> {
+    private getAllSections(visible: boolean): Array<ActivitySection> {
```

Comment:
I think `onlyVisible` would be a better name

Reply:
OKI. Just a moment